build-tools/aminclude/autodocs-ycp.ami

```diff
@@ -48,9 +48,21 @@ index.html: $(AUTODOCS_YCP) $(AUTODOCS_PM) $(AUTODOCS_RB)
 		echo "<a href='$$b.html'>$${b/__/::}</a>" > $$b.mod.html; \
 	done
 	if [ -n "$(AUTODOCS_RB)" ]; then \
-          if which $(YARD) 2>/dev/null; then \
-            $(YARD) doc --title "@RPMNAME@ -- Development Documentation" $(YARD_OPTIONS) $(AUTODOCS_RB);\
-          else \
-            echo "yard not available, skipping"; \
-          fi; \
-        fi
+		if which $(YARD) 2>/dev/null; then \
+			$(YARD) doc --title "@RPMNAME@ -- Development Documentation" $(YARD_OPTIONS) $(AUTODOCS_RB);\
+		else \
+			echo "yard not available, skipping"; \
+			if [ ! -f index.html ]; then \
+				echo '<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">\
```

Comment:
Um, HTML4? I'd prefer HTML5, it's much more simpler, see this [HTML5 Example](http://www.w3schools.com/html/html5_intro.asp).